mslib/msui/multiple_flightpath_dockwidget.py

```diff
@@ -177,16 +178,21 @@ def __init__(self, parent=None, view=None, listFlightTracks=None,
 
     @QtCore.pyqtSlot()
     def logout(self):
+        # A disconnect can be done only once, regardless of the docking widget usage frequency.
         if self.operations is not None:
             self.operations.logout_mscolab()
-        self.ui.signal_listFlighttrack_doubleClicked.disconnect()
-        self.ui.signal_permission_revoked.disconnect()
-        self.ui.signal_render_new_permission.disconnect()
-        self.operations = None
-        self.flighttrack_list = True
-        self.operation_list = False
-        for idx in range(len(self.obb)):
-            del self.obb[idx]
+        try:
+            self.ui.signal_listFlighttrack_doubleClicked.disconnect()
+            self.ui.signal_permission_revoked.disconnect()
+            self.ui.signal_render_new_permission.disconnect()
```

Comment:
Same as below; I think this doesn't serve its intended purpose, which might be the underlying bug.